### DISCUSSION TITLE
Ask how ngrinder does mixed scenarios

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#816] [Q&A]
Ask how ngrinder does mixed scenarios. The return value of the previous interface is the input parameter of the next interface, and TPS can be distinguished
@junoyoon @Mavlarn @maoyubin

**Comment by donggyu04** [comment]
@zhougy1989 

I think it is impossible. each function in the script works synchronously. although each function can be measured as a separate TPS.
But, the function executed earlier waits for the latter functions, so the measured TPS in the preceding function is not accurate.

measure TPS in separate tests for each function, or assume the entire flow as one transaction.
